### PR TITLE
US-6: Implement sidebar structure

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 1]
```diff
@@ -10,50 +10,72 @@
...
+                        <portlet>
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceAdministrationNavigation</portlet-ref>
+                        </portlet>
+                        <access-permissions>*:/platform/users</access-permissions>
+                        <show-info-bar>false</show-info-bar>
+                    </portlet-application>
+                </container>
+
+                <container id="DWSpaceNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: DWSpacesNavigationPortlet

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 2]
```diff
@@ -10,50 +10,72 @@
...
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceSpacesNavigation</portlet-ref>
                         </portlet>
                         <access-permissions>*:/platform/users</access-permissions>
                         <show-info-bar>false</show-info-bar>
                     </portlet-application>
                 </container>
-                <container id="SpaceNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+
+                <container id="DWAccountNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: DWMyAccountPortlet

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 3]
```diff
@@ -10,50 +10,72 @@
...
                     <access-permissions>*:/platform/users</access-permissions>
                     <portlet-application>
                         <portlet>
-                            <application-ref>platformNavigation</application-ref>
-                            <portlet-ref>UISpaceNavigationPortlet</portlet-ref>
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceProfileNavigation</portlet-ref>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: digitalWorkplaceMyAccount

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 4]
```diff
@@ -10,50 +10,72 @@
...
                         <access-permissions>*:/platform/users</access-permissions>
                         <show-info-bar>false</show-info-bar>
                     </portlet-application>
                 </container>
-                <container id="bottom-leftNavigation-container" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
-				          <name>bottom-leftNavigation-container</name>
-				          <access-permissions>*:/platform/users</access-permissions>
-				          <factory-id>addonContainer</factory-id>
+
+                <container id="DWSettingsNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: DWMySettingsPortlet

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 5]
```diff
@@ -10,50 +10,72 @@
...
+                    <access-permissions>*:/platform/users</access-permissions>
+                    <portlet-application>
+                        <portlet>
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceSettingsNavigation</portlet-ref>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: digitalWorkplaceMySettings

File: digital-workplace-webapps/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 6]
```diff
@@ -291,5 +291,135 @@
     </depends>
   </module>
 
+  <portlet>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/gatein-resources.xml]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/WEB-INF/portlet.xml [hunk 7]
```diff
@@ -151,4 +151,81 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http:/
 		</portlet-info>
 	</portlet>
 
+	<!-- ************************* Sidebar portlets ********************************* -->
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/WEB-INF/portlet.xml]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/administrationNavigation.html [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/administrationNavigation.html]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/companyNavigation.html [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/companyNavigation.html]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/profileNavigation.html [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/profileNavigation.html]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIHomeLinkContainer.gtmpl [hunk 11]
```diff
@@ -0,0 +1,10 @@
+<%
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIHomeLinkContainer.gtmpl]: Same remark as above

File: digital-workplace-webapps/src/main/webapp/settingsNavigation.html [hunk 12]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/settingsNavigation.html]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/spacesNavigation.html [hunk 13]
```diff
@@ -0,0 +1,9 @@
+<div class="VuetifyApp">
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/spacesNavigation.html]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/components/AdministrationApp.vue [hunk 14]
```diff
@@ -0,0 +1,8 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/components/AdministrationApp.vue]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/main.js [hunk 15]
```diff
@@ -0,0 +1,29 @@
+import adminitrationNavigationApp from './components/AdministrationApp.vue';
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/main.js]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/components/CompanyNavigationApp.vue [hunk 16]
```diff
@@ -0,0 +1,9 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/components/CompanyNavigationApp.vue]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/main.js [hunk 17]
```diff
@@ -0,0 +1,29 @@
+import CompanyNavigationApp from './components/CompanyNavigationApp.vue';
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/main.js]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/components/ProfileNavigationApp.vue [hunk 18]
```diff
@@ -0,0 +1,9 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/components/ProfileNavigationApp.vue]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/main.js [hunk 19]
```diff
@@ -0,0 +1,29 @@
+import ProfileNavigationApp from './components/ProfileNavigationApp.vue';
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/main.js]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/components/SettingsNavigationApp.vue [hunk 20]
```diff
@@ -0,0 +1,9 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/components/SettingsNavigationApp.vue]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/main.js [hunk 21]
```diff
@@ -0,0 +1,29 @@
+import SettingsNavigationApp from './components/SettingsNavigationApp.vue';
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/main.js]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/components/SpacesNavigationApp.vue [hunk 22]
```diff
@@ -0,0 +1,9 @@
+<template>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/components/SpacesNavigationApp.vue]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/main.js [hunk 23]
```diff
@@ -0,0 +1,29 @@
+import SpacesNavigationApp from './components/SpacesNavigationApp.vue';
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/main.js]: Update this file according to the above remarks

File: digital-workplace-webapps/webpack.common.js [hunk 24]
```diff
@@ -15,7 +15,12 @@ let config = {
     events: './src/main/webapp/vue-app/events/main.js',
     sharedDocuments: './src/main/webapp/vue-app/sharedDocuments/main.js',
     recentDocuments: './src/main/webapp/vue-app/recentDocuments/main.js',
-    favoriteDocuments: './src/main/webapp/vue-app/favoriteDocuments/main.js'
+    favoriteDocuments: './src/main/webapp/vue-app/favoriteDocuments/main.js',
```
Comment (azayati) [inline digital-workplace-webapps/webpack.common.js]: Update this file according to the above remarks

File: digital-workplace-webapps/src/main/webapp/css/main.less [hunk 25]
```diff
@@ -478,3 +506,9 @@
     line-height: 1.75rem;  text-transform: capitalize;
   }
 }
+
+/*----------------------Just for test ----------------------------*/
+
+#WLCFooter {
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/css/main.less]: Done!

File: digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIDWPageContainer.gtmpl [hunk 26]
```diff
@@ -0,0 +1,107 @@
+ <%
+	import org.exoplatform.commons.utils.ExpressionUtil;
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIDWPageContainer.gtmpl]: Done!

File: digital-workplace-webapps/src/main/webapp/groovy/portal/webui/workspace/UIWorkingWorkspace.gtmpl [hunk 27]
```diff
@@ -0,0 +1,83 @@
+<%
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/groovy/portal/webui/workspace/UIWorkingWorkspace.gtmpl]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml (hunk 3 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 28]
```diff
@@ -10,50 +10,72 @@
...
                 <access-permissions>*:/platform/users</access-permissions>
-                <container id="LeftNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+                <container id="DWLeftNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
                     <access-permissions>*:/platform/users</access-permissions>
                     <portlet-application>
                         <portlet>
-                            <application-ref>platformNavigation</application-ref>
-                            <portlet-ref>UICompanyNavigationPortlet</portlet-ref>
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceCompanyNavigation</portlet-ref>
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml [hunk 29]
```diff
@@ -10,50 +10,72 @@
...
                     </portlet-application>
                 </container>
-                <container id="GroupsNavigationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+
+                <container id="DWAdministrationPortlet" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+                    <access-permissions>*:/platform/users</access-permissions>
+                    <portlet-application>
+                        <portlet>
+                            <application-ref>digital-workplace</application-ref>
+                            <portlet-ref>digitalWorkplaceAdministrationNavigation</portlet-ref>
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml (hunk 5 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml (hunk 4 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 6 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/gatein-resources.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/portlet.xml (hunk 7 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/portlet.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIHomeLinkContainer.gtmpl (hunk 11 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/groovy/portal/webui/container/UIHomeLinkContainer.gtmpl]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/components/SettingsNavigationApp.vue (hunk 20 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/components/SettingsNavigationApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/main.js (hunk 21 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/settingsNavigation/main.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/components/SpacesNavigationApp.vue (hunk 22 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/components/SpacesNavigationApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/main.js (hunk 23 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/spacesNavigation/main.js]: Done!

File: digital-workplace-webapps/webpack.common.js (hunk 24 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/webpack.common.js]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml (hunk 1 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml (hunk 2 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/WEB-INF/conf/digital-workplace/portal/dw/portal.xml]: Done!

File: digital-workplace-webapps/src/main/webapp/administrationNavigation.html (hunk 8 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/administrationNavigation.html]: Done!

File: digital-workplace-webapps/src/main/webapp/companyNavigation.html (hunk 9 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/companyNavigation.html]: Done!

File: digital-workplace-webapps/src/main/webapp/profileNavigation.html (hunk 10 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/profileNavigation.html]: Done!

File: digital-workplace-webapps/src/main/webapp/settingsNavigation.html (hunk 12 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/settingsNavigation.html]: Done!

File: digital-workplace-webapps/src/main/webapp/spacesNavigation.html (hunk 13 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/spacesNavigation.html]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/components/AdministrationApp.vue (hunk 14 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/components/AdministrationApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/main.js (hunk 15 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/administrationNavigation/main.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/components/CompanyNavigationApp.vue (hunk 16 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/components/CompanyNavigationApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/main.js (hunk 17 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/companyNavigation/main.js]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/components/ProfileNavigationApp.vue (hunk 18 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/components/ProfileNavigationApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/main.js (hunk 19 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/profileNavigation/main.js]: Done!

**Pull request merged**